### PR TITLE
Add docs about updating to V8

### PR DESCRIPTION
**Pull request by matyasf**

### CODE REVIEW

File: docs/guides/button-upgrade-guide.md [hunk 1]
```diff
@@ -1,30 +1,32 @@
 ---
 title: Button Upgrade for Version 8.0
 category: Guides
-order: 4
+order: 2
```
Comment (matyasf) [inline docs/guides/button-upgrade-guide.md:4]: the only change here

File: docs/guides/focus-management.md [hunk 2]
```diff
@@ -1,29 +1,37 @@
 ---
 title: Focus Management
 category: Guides
-order: 1
+order: 3
```
Comment (matyasf) [inline docs/guides/focus-management.md:4]: the only change here

File: docs/guides/v7-upgrade-guide.md [hunk 3]
```diff
@@ -1,19 +1,22 @@
 ---
 title: Upgrade Guide for Version 7.0
 category: Guides
-order: 1
+order: 4
```
Comment (matyasf) [inline docs/guides/v7-upgrade-guide.md:4]: the only change here. I guess we can delete this doc btw :)

File: packages/ui-docs-client/src/Hero/index.js [hunk 4]
```diff
@@ -485,7 +485,7 @@ class Hero extends Component {
                       withBorder={false}
                       focusColor="inverse"
                       color="success"
-                      href="#v7-upgrade-guide"
+                      href="#v8-upgrade-guide"
                       size={bigScreen ? 'large' : 'medium'}
                       margin="0 x-small x-small 0"
                     >
```
Comment (ke1k0) [inline packages/ui-docs-client/src/Hero/index.js:491]: 8.0 Upgrade Guide (button label)

File: docs/guides/v7-upgrade-guide.md [hunk 5]
```diff
@@ -1,273 +0,0 @@
----
-title: Upgrade Guide for Version 7.0
```
[the remark this answers is not in the document]
Comment (matyasf) [reply docs/guides/v7-upgrade-guide.md:2]: If they want to upgrade to V7 they need to look at the legacy documentation. It contains all the other stuff too (e.g. deprecations)

[the remark this answers is not in the document]
Comment (ke1k0) [reply docs/guides/v7-upgrade-guide.md:2]: I just thought that when we release v9, and replace the v7 docs with v8 as "legacy", these info will be lost :) And if we are planning to release majors more often, we need to find a solution for it.

**Review by Brailor — APPROVED** [review]

**Review by ke1k0 — APPROVED** [review]

**Pull request merged**